boards/laird_connect/bl654_dvk/bl654_dvk_nrf52840_pa_defconfig

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Limit max power for FCC compliance with power amplifier
+CONFIG_BT_CTLR_TX_PWR_MINUS_4=y
```

Comment:
Might want to move this to `Kconfig.defconfig` so that it only sets it as a default only when bluetooth is enabled

Reply:
This did not get set when building the hello_world sample app for this variant, for example.

But I am going to move it.